playwright.config.ts

```diff
@@ -10,7 +10,9 @@ import { defineConfig, devices } from '@playwright/test';
  * See https://playwright.dev/docs/test-configuration.
  */
 export default defineConfig({
-  timeout: 50000,
```

Comment:
Default timeout is 30000ms, if you want to decrease it, consider that it should be more then 1000ms (1s), because in most cases test will fail if element presence is loading more then 1s.